mirror-godot-app/script/gd/gdscript_instance.gd

```diff
@@ -62,6 +62,8 @@ static var _SCRIPT_TEXT_DENYLIST: Array[RegEx] = [
 	RegEx.create_from_string("\\bpreload\\b"),
 	RegEx.create_from_string("\\bOS\\b"),
 	RegEx.create_from_string("\\bResourceLoader\\b"),
+	RegEx.create_from_string("\\FileAccess\\b"),
+	RegEx.create_from_string("\\DirAccess\\b")
```

Comment:
`\\b` escapes to `\b` which is a word boundary.

```suggestion
	RegEx.create_from_string("\\bFileAccess\\b"),
	RegEx.create_from_string("\\bDirAccess\\b")
```

Reply:
Thank you @aaronfranke!